### ISSUE TITLE
`ter-indent` rule flags lines incorrectly for chained functions whose args span multiple lines

### ISSUE THREAD

**Issue by wesm87** [buzinas/tslint-eslint-rules#333]
I'm having trouble with the `ter-indent` rule giving me false positives in cases where I have a number of chained functions spread across multiple lines, and one of those functions has its arguments spread across multiple lines. For example:

```js
startChainedMethods()
  .foo()
  .bar()
  .baz(
    'hello world',
    {
      data: { ... },
    },
    callback,
  );
```

In the example above lines 5-9 would be flagged as having 2 extra spaces, presumably because it's counting from the first character of `startChainedMethods()` instead of the first character of `.baz()`. The same code works fine if I put the chained function calls on a single line like so:

```js
startChainedMethods().foo().bar().baz(
  'hello world',
  {
    data: { ... },
  },
  callback,
);
```

#### Versions

```
$ tslint --version
5.9.1
$ tsc --version
Version 2.7.2
```

#### Configuration 

The TSLint configuration is in a custom package and the `ter-indent` rule is configured like so:

```js
'ter-indent': [true, 2, {
  SwitchCase: 1,
  VariableDeclarator: 1,
  outerIIFEBody: 1,
  MemberExpression: 1,
  FunctionDeclaration: {
    parameters: 1,
    body: 1,
  },
  FunctionExpression: {
    parameters: 1,
    body: 1,
  },
  CallExpression: {
    arguments: 1,
  },
}]
```

**Comment by jmlopez-rod** [comment]
This... really bothers me. I'd like to fix it but there is very limited amount of time. I'm going to kindly ask to some kind stranger to help us out here in the mean time.
